src/azure-cli/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+2.0.58
+++++++
```

Comment:
i don't think version upgrade to a future one is needed now. Has anything changed recently?